css/about.css

```diff
@@ -0,0 +1,199 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0 auto;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+
+
+
+
+.container{
+    width: 800px;
+    margin: 3 auto;
+    margin-bottom: 20px;
+    border-bottom: 1px dashed black;
+    padding:2%;
+}
+
+.whole-page{
+    margin:0 auto !important;
+    width:100%;
+}
+
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+
+.navigation{
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 5%;
+}
+
+a{
+    text-decoration: none;
+    color: black;
+}
+
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+* {
+    box-sizing: border-box;
+    margin:0 auto;
+    max-width: 800px;
+    
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-size: 18px;
+    margin-bottom: 15px;
+    font-family: 'Rubik', sans-serif;
+    margin:0 auto;
+}
+
+.top-content{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+   
+}
+
+.top-content__left{
```

Comment:
as all of your `top-content_left`, `top-content_right`, `middle-content_left`, `middle-content_right` classes have the same styling you can apply DRY principles by taking one of two approaches.

1. Rather than having separate classes you can just give all the divs the same class name `content_container` and have your styling as such:

```css
.content__container{
    display: flex;
    flex-direction: column;
    width: 45%;
    justify-content: center;
    padding: 5%;
    margin:0 auto;
    margin-bottom: 20px;
    border-bottom: 1px dashed black;
}
```

2. You can apply the same css styling to both classes in at one like so:
```css
.top-content__left, .top-content_right,
.middle-content_left, .middle-content_right {
    display: flex;
    flex-direction: column;
    width: 45%;
    justify-content: center;
    padding: 5%;
    margin:0 auto;
    margin-bottom: 20px;
    border-bottom: 1px dashed black;
}
```